packages/nimble-components/src/select/template.ts

```diff
@@ -58,6 +58,7 @@ SelectOptions
         aria-disabled="${x => x.ariaDisabled}"
         aria-expanded="${x => x.ariaExpanded}"
         aria-haspopup="${x => (x.collapsible ? 'listbox' : null)}"
+        aria-label="${x => x.labelContent}"
```

Comment:
I would have expected the `Combobox` to need this as well.

Reply:
There's still a bit of logic present to produce the `labelContent` string. I presume we've concluded that the results produced by that logic are unneeded for the `Combobox`?

Reply:
The logic of `labelContent` is just used to compute the value for `aria-label`, whereas combobox will just use the label text directly for that.

In terms of the computed `labelContent` value vs. what the label shows visually, I checked that both handle plain text and  `span` elements. One minor difference I see is that `<span>A B C</span><span>D E F</span>` shows `A B CD E F` visually in the label (vs `labelContent` of `A B C D E F`). But the way the label is shown matches how other existing controls (number field/ text field/etc) show it, so I think it's OK.